### PR TITLE
DSS API: Security Spec Changes

### PR DESCRIPTION
**Pull request by amarjandu**
allows security audience to be retrievable from swagger
removes reference to dcp from definitions. 

Related to https://github.com/DataBiosphere/data-store-cli/pull/8

### CODE REVIEW

**Review by chmreid — APPROVED** [review]
LGTM

We can revisit the `auth.ucsc.ucsc-cgp-redwood.org` URL later, if we want to add stage to the subdomain/have multiple deployments

File: dss-api.yml [hunk 1]
```diff
@@ -119,11 +119,12 @@ securityDefinitions:
       - '{{OIDC_EMAIL_CLAIM}}': the email of the requester
     type: oauth2
     flow: implicit
-    authorizationUrl: https://auth.data.humancellatlas.org/authorize
+    authorizationUrl: https://auth.ucsc.ucsc-cgp-redwood.org/authorize
```
Comment (chmreid) [inline dss-api.yml]: You use `{{API_DOMAIN_NAME}}` below, i.e., environment variable substitution. Should this also use an environment variable for the authorization URL?

Comment (amarjandu) [reply dss-api.yml]: its going to be a fixed value (most of the time), but i agree lets see if this can be extracted from env var.

Comment (amarjandu) [reply dss-api.yml]: it works!~

**Pull request merged**